### ISSUE TITLE
AttributeError: 'Video' object has no attribute 'time_vid_start'

### ISSUE THREAD

**Issue by talhaanwarch** [phuselab/pyVHR#9]
I am running on windows 10, python 3.8 and getting this error with basic demo file
```
AttributeError                            Traceback (most recent call last)
<ipython-input-8-d1815841500d> in <module>
     15 #       4 - display errors
     16 #       (use also combinations, e.g. verb=21, verb=321)
---> 17 result = test.start(outFilename='sampleExp.h5', verb=1)

D:\rPPG\paper rppg\pyVHR-master\pyenv\lib\site-packages\pyvhr-0.0.3-py3.8.egg\pyVHR\analysis\testsuite.py in start(self, saveResults, outFilename, verb)
     61                 videoFilename = dataset.getVideoFilename(v)
     62                 video = Video(videoFilename, verb)
---> 63                 video.getCroppedFaces(detector=self.videodict['detector'],
     64                                       extractor=self.videodict['extractor'])
     65                 etime = float(self.videodict['endTime'])

D:\rPPG\paper rppg\pyVHR-master\pyenv\lib\site-packages\pyvhr-0.0.3-py3.8.egg\pyVHR\signals\video.py in getCroppedFaces(self, detector, extractor)
    107             # -- extract faces and resize
    108             print('\n\n' + detector + '\n\n')
--> 109             self.__extractFace(video, method=detector)
    110 
    111             # -- store cropped faces on disk

D:\rPPG\paper rppg\pyVHR-master\pyenv\lib\site-packages\pyvhr-0.0.3-py3.8.egg\pyVHR\signals\video.py in __extractFace(self, video, method, t_downsample_rate)
    541                 new_time_vid_start = skipped_frames / self.frameRate
    542 
--> 543                 if new_time_vid_start > self.time_vid_start:
    544                     self.time_vid_start = new_time_vid_start
    545                     print("\tVideo now starts at " + str(self.time_vid_start) + " seconds\n")

AttributeError: 'Video' object has no attribute 'time_vid_start'
```

**Comment by talhaanwarch** [comment]
One way to handle is to comment out these three lines before installing the package

**Comment by vcuculo** [comment]
Solved by #10
